.github/workflows/ci-test-playwright.yml

```
@@ -0,0 +1,198 @@
+        ports:
+          - 6379:6379
+      mongo:
+        image: mongo
+        ports:
+          - 27017:27017
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.pr != 0 && format('refs/pull/{0}/merge', inputs.pr) || '' }}
```

actionlint (1.7.12)

[error] property "pr" is not defined in object type {docker_image_name: string; feature: string; flag_overrides: string; project: string}

(expression)